app/src/examples/AnimatedSensorGyroscopeExample.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Pressable, SafeAreaView, StyleSheet, Text, View } from 'react-native';
+import Animated, {
+  SensorType,
+  useAnimatedSensor,
+  useAnimatedStyle,
+  useSharedValue,
+  withSpring,
+} from 'react-native-reanimated';
+
+function clampWorklet(num: number, min: number, max: number) {
+  'worklet';
+  return Math.min(Math.max(num, min), max);
+}
+
+const OFFSET_X = 100; //in px
+const OFFSET_Y = 100; //in px
+const OFFSET_Z = 180; //in degrees
```

Comment:
btw what about renaming these constants to `MAX_OFFSET_*` or `THRESHOLD_*`?
```suggestion
const OFFSET_X = 100; // in px
const OFFSET_Y = 100; // in px
const OFFSET_Z = 180; // in degrees
```